lib/supavisor_web/endpoint.ex

```diff
@@ -10,6 +10,14 @@ defmodule SupavisorWeb.Endpoint do
     signing_salt: "zJOrGxcM"
   ]
 
+  socket "/socket/vercel", SupavisorWeb.WsProxy,
+    websocket: [path: "/"],
+    longpoll: false
+
+  socket "/v2", SupavisorWeb.WsProxy,
```

Comment:
Also this we should handle in the API gateway and just route /v2 to /socket.

But it's okay in here for now for testing until we get everything else setup.